### ISSUE TITLE
Docs image should all be properly described for accessibility

### ISSUE THREAD

**Issue by KevinMGranger** [konveyor/pelorus#341]
Markdown images use the following syntax: `![alt text](image url)`. The alt text will be used as the `alt` attribute on the resulting `img` element.

[Images should have a meaningful alt-text](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#accessibility_concerns) for the sake of accessibility.

For someone using a screen reader, something like `Value Delivery Cycle` is less useful than a description of what that cycle says in its text.
